### PR TITLE
Project Responsive Web Design milestone 2

### PR DESCRIPTION
**Pull request by JAAR91**
Project Responsive Web Design 2

On this project with complete previous clone project for website:

https://perma.cc/M5ZV-Q2D6

The project was mean to add:

-Latest News
-Categories
-Latest Deals

This project was made using CSS and HTML, also the queries were used to make it responsive for cellphones, tablets, and desktops.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,1571 @@
...
+}
+
+.deal1,
+.deal2,
+.deal3,
+.deal4,
+.deal5,
+.deal6,
+.deal7,
+.deal8 {
```
Comment (wathigo) [inline styles.css]: Let's keep the [code clean](https://www.w3schools.com/html/html5_syntax.asp). We can assign a class name, let's day `deal` to all deals, then style as a class.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,1571 @@
...
+  }
+
+  .news1 h4:hover,
+  .news2 h4:hover,
+  .news3 h4:hover,
+  .news4 h4:hover,
+  .news5 h4:hover,
+  .news6 h4:hover,
+  .news7 h4:hover,
+  .news8 h4:hover  {
```
Comment (wathigo) [inline styles.css]: Let's keep our code clean by using a single class name instead.

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this milestone.
Let's fix a few issues before we move on.
See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Requires changes :yellow_circle: 
### Hello  team :wave: 

#### Thanks for your submission.  

### Please work on the suggested requirements made by the previous TSE. Use one class name for elements with the same style, that is the purpose of CSS classes and it will help you minimize your CSS code. 
> eg: instead of .deal1, .deal2, .deal3 classes use one .deal class for all of them.

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Review by SmithVyne — APPROVED** [review]
### Project Approved 🎖️ 🎖️ 
Hello, @jaarkira congratulations on making all the required changes pointed out by the previous TSEs. Your project is now approved. You can now merge this branch and move on in the curriculum.

You can reach out to me on slack, **@SmithVyne** if you have any other questions.

**Pull request merged**